{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/conftest.py

```diff
@@ -1,12 +1,37 @@
 # pylint: disable=redefined-outer-name
 # flake8: noqa: E800
 import os
-from collections.abc import Generator
+from typing import Generator
 
 import pytest
 from playwright.sync_api import BrowserContext, ConsoleMessage, Error, Page, Playwright
 
 
+def pytest_collection_modifyitems(items: list[pytest.Item]) -> None:
```

Comment:
Is this a special hook function used by pytest? It's not referenced anywhere else in the PR, and I wasn't quite I sure understood the docs describing it, so I just wanted to flag it

Reply:
Yeah, it's a hook function when defined in a `conftest.py` like it is here.